src/rules/pointfree.js

```diff
@@ -0,0 +1,57 @@
+import { isIdentifier, isCallExpression } from "utils"
+
+export default {
+  meta: {
+    docs: {
+      description: "Prefer pointfree definitions of functions",
+      category: "General",
+      recommended: false,
+    },
+    fixable: null,
+    schema: [],
```

Comment:
Ultimately might make sense for this to accept a blacklist option (for names of functions that you know accept multiple arguments but you only want to call with one so can't safely use in a pointfree style if the caller might supply multiple arguments eg `parseInt()`)

Speaking of which, `parseInt()` should probably be blacklisted by default?

Reply:
Right, I forgot about some of the discussion we had about this. I added a blacklist option, and included a list of default blacklisted functions (which is just `parseInt` for now).

Reply:
Ok it might be more typical to have the blacklist supplied as an option directly to this rule since it only applies to this one rule, eg:
```
"blackbird/pointfree": ["error", {"blacklist": [...]}]
```

But I don't mind it as a setting at least for now